libraries/AP_HAL/UARTDriver.h

```diff
@@ -96,6 +96,7 @@ class AP_HAL::UARTDriver : public AP_HAL::BetterStream {
         OPTION_MAVLINK_NO_FORWARD = (1U<<10), // don't forward MAVLink data to or from this device
         OPTION_NOFIFO             = (1U<<11), // disable hardware FIFO
         OPTION_NOSTREAMOVERRIDE   = (1U<<12), // don't allow GCS to override streamrates
+        OPTION_MAVLINK_FORWARD_BAD_CRC = (1U<<13), // forward mavlink packets that don't pass CRC
```

Comment:
Separate option for bad signature?

Reply:
Perhaps as a future enhancement.

Reply:
Had a quick look at this, and it probably does deserve a bit of discussion before we merge this.  Adding the extra bit now would be easy, adding later might be problematic.